optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -76,10 +76,24 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
             // Abbreviate to 2 decimals
             // We don't use DecimalFormat to abbreviate because it's written locale insensitive (like java literals)
             int dotIndex = abbreviated.lastIndexOf('.');
-            if (dotIndex >= 0 && dotIndex + 3 < abbreviated.length()) {
-                abbreviated = abbreviated.substring(0, dotIndex + 3);
+            StringBuilder result = new StringBuilder().append(abbreviated.substring(0, dotIndex < 0 ? abbreviated.length() : dotIndex));
```

Comment:
Nitpick: StringBuilder has this method:
  public StringBuilder append(CharSequence s, int start, int end)

This avoids the substring()'s string creation and is trivially, completely unnoticeable faster.